server/index.js

```diff
@@ -1,18 +1,30 @@
 const axios = require('axios');
 const bodyParser = require('body-parser');
+const cors = require('cors');
 const db = require('../database/index.js');
 const express = require('express');
 
 const app = express();
 const port = 3003;
 
+
+
 app.listen(port, () => {
   console.log(`Server listening at http://127.0.0.1:${port}`);
 });
+
+const allowedOrigins = [
+  'http://localhost:3000',
+  'http://localhost:5500',
+];
+app.use(cors());
+app.use(cors({ origin: allowedOrigins }));
```

Comment:
Oh right on, so you're adding enabling your Proxy to access every other service via this origins array. That's super cool.